VERSION

```diff
@@ -1 +0,0 @@
-2.0.0
```

Comment:
This file isn't used anywhere and it just going to stay outdated if we don't remove it